Application Actions/Preview/1.0/default.json

```diff
@@ -0,0 +1,9 @@
+{
+  "RotateLeft"  : "Scroll Down",
+  "RotateRight" : "Scroll Up",
```

Comment:
But all other applications in this repo have rotate left -> scroll down. You think we should change it for all of them?